src/transformers/models/altclip/processing_altclip.py

```diff
@@ -95,13 +117,24 @@ def __call__(self, text=None, images=None, return_tensors=None, **kwargs):
         """
 
         if text is None and images is None:
-            raise ValueError("You have to specify either text or images. Both cannot be none.")
+            raise ValueError("You must specify either text or images.")
 
-        if text is not None:
-            encoding = self.tokenizer(text, return_tensors=return_tensors, **kwargs)
+        if text is None and images is None:
+            raise ValueError("You must specify either text or images.")
+        output_kwargs = self._merge_kwargs(
+            AltClipProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
```

Comment:
Do you think we should have an equivalent `init_kwargs` for the image processors and feature extractors for consistency? Or better to leave just in one place?

Reply:
currently the image processors and other extractors don't need this to work (this is mostly for fast tokenizers), not opposed to including it as well but it seems to be more a tokenizers thing?

Reply:
OK, let's leave for now!